Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -711,6 +728,24 @@ bool EventsRefactorer::SearchStringInConditions(
   return false;
 }
 
+bool EventsRefactorer::SearchStringInComments(gd::ObjectsContainer& project,

```

Comment:
Rename this to `SearchStringInEvents`
There is no notion of "comments" here. The strings might be in the future strings coming from somewhere else... sure for now only comments have strings, but that's a coincidence :)